src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -254,7 +254,7 @@ namespace {
 // Channel arg pointer vtable for GRPC_ARG_CLIENT_CHANNEL.
 void* ClientChannelArgCopy(void* p) { return p; }
 void ClientChannelArgDestroy(void* /*p*/) {}
-int ClientChannelArgCmp(void* p, void* q) { return GPR_ICMP(p, q); }
+int ClientChannelArgCmp(void* p, void* q) { return grpc_core::icmp(p, q); }
```

Comment:
No need for `grpc_core::`, since we're already in that namespace here.

Same thing for all code in the client_channel tree.

Reply:
done